lib/src/page_list_viewport_gestures.dart

```diff
@@ -288,35 +305,190 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   }
 }
 
-class DeprecatedPanAndScaleVelocityTracker {
-  /// The maximum velocity a gesture can have and still be considered a viewport
-  /// re-adjustment. Used in tandem with the [kViewportReAdjustmentMinTranslationDistance]
-  /// to determine if the user is re-adjusting the viewport. Viewport re-adjustments
-  /// result in the momentum simulation to be aborted.
-  static double kViewportReAdjustmentMaxVelocity = 600;
+/// Definiton for gestures' translation distance and velocity categories.
+///
+/// Distances are tiny, small, and large.
+/// Velocities are slow, normal, and fast.
+/// These categories are used to individually define ballistic simulation behavior
+/// across a variety of scrolling situations.
+class GestureThresholdsAndScales {
+  /// Definition for a tiny distance in pixels.
```

Comment:
For each of these properties, please change the first line to describe them as a "max" because that's how you describe these values in the template section below.

`/// The maximum distance for a motion to be categorized as "tiny".`